### PR TITLE
Debug info removed and comments added

### PR DESCRIPTION
**Pull request by mystcb**
Couple of changes in this PR:

- Debugging info was removed from some of the JS files, as a bit of cleanup - but also to remove some alerts which were out of place!
- Added comments to the `submission.js` file to make it clear what is happening at each stage
- Added a reload into the `submission.js` file to allow quick rediscovery of tags on the main page
- Made the S3 bucket creation optional and updated the `README.md` to reflect the changes

Bit of an update ;)

### CODE REVIEW

File: website/js/submission.js [hunk 1]
```diff
@@ -1,23 +1,32 @@
+// Prevents the standard form submission from POSTing and allows the JS below to work
+// TODO: CB Is this still required? I spot its duplicated elsewhere!
 $("#tagForm").submit(function(e){
     e.preventDefault();
-    alert("Hello"); 
 })
```
Comment (Yamakiroshi) [inline website/js/submission.js]: You can probably remove this, as you say there is a duplicate further down in the page and that should be able to handle the default processes of the submit.  (as long as the tagForm submit box is there on document ready) which it should be.

File: website/js/submission.js [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
             method: "POST", 
             url: "https://<API-GATEWAY-ADDRESS>/<STAGE>/putTagsForWishTree",
             data: JSON.stringify(jsonObject),
             dataType: 'json',
             crossDomain: true,
         }).done(function (msg) {
-            console.log(msg);
-            alert( "Tag Saved: " + msg);
+            // If successful, then reload the page so that the new tag can be loaded in
+            location.reload();
```
Comment (Yamakiroshi) [inline website/js/submission.js:23]: This a good addition, automatically refreshing when submitted.

File: website/js/sketch.js [hunk 3]
```diff
@@ -1,6 +1,5 @@
 function preload () {
   loadedTags = loadJSON("https://<API-GATEWAY-ADDRESS>/<STAGE>/getTagsForWishTree");
-  //console.log(loadedTags);
```
Comment (Yamakiroshi) [inline website/js/sketch.js:3]: The removal of the comments / debug messages in this one are good and make sense!

**Review by Yamakiroshi — COMMENTED** [review]
All good to me with the improvement and making the S3 optional is a nice addition! You can remove that duplicated bit of code and then it's all thumbs up from me :D

File: website/js/submission.js (hunk 1 above)
Comment (mystcb) [reply website/js/submission.js]: Technically this update does remove it ;)

**Comment by mystcb** [comment]
Removed the extra bit of code. resolved the comments! 

LMK if this is OK!

**Review by Yamakiroshi — APPROVED** [review]
LGTM Merging it in :D

**Pull request merged**